teps/0079-tekton-catalog-support-tiers.md

```diff
@@ -8,6 +8,7 @@ authors:
 - '@jerop'
 - '@vdemeester'
 - '@vinamra28'
+- '@chmouel'
```

Comment:
@chmouel added you as a co-author in this TEP - please let me know if you'd prefer not to be added 😀

Reply:
SGTM :)

/lgtm